### DISCUSSION TITLE
Blocked Account

### DISCUSSION THREAD

**Discussion by 0xAizen** [vercel/community#4274] [Account Issue] [answered]
Trying to login and update my payment for my domain. Is there any reason why this happened?

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with login. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy. That will get you in contact with the team that can help with login issues.

**Reply by 0xAizen** [reply]
I have received 0 emails from Vercel about the error. I am just trying to recover my domain, I've emailed already. I have received a bunch of renewal emails but I didn't notice that my card was still not changed to the correct one.

**Reply by amyegan** [reply]
Would you mind sharing the case number? Since domain renewal is part of your problem, I want to make sure the ticket was assigned to the right team 🙏

**Reply by 0xAizen** [reply]
> Would you mind sharing the case number? Since domain renewal is part of your problem, I want to make sure the ticket was assigned to the right team 🙏 

Here is the case number: 00161888

Thank you so much, I wasn't expecting my domain to go down and I'm missing some important emails.

**Comment by amyegan** [comment]
It looks like your ticket was updated and your domain was renewed, so I'm closing this discussion now 🙂
